bin/docker-run.sh

```diff
@@ -2,4 +2,4 @@
 
 set -eux
 
-docker run -e POSTGRES_PASSWORD=foo --rm -p 5432:5432 --volume "$PWD/share/dbs:/share/dbs" mmpg
+docker run -e POSTGRES_PASSWORD=foo --rm --shm-size=1g -p 5432:5432 --volume "$PWD/share/dbs:/share/dbs" mmpg
```

Comment:
Could you add a comment explaining why this change is necessary?

Reply:
ah, i don't think that is the change that was necessary -- let me take a look